Dockerfile

```diff
@@ -32,7 +32,7 @@ RUN groupadd -g ${BAZEL_USER_ID} -r ${BAZEL_USER} \
  && chmod +x ${ECR_LOGIN_PATH}
 
 COPY --from=plugin /go/bin/drone-bazelisk-ecr /usr/local/bin/drone-bazelisk-ecr
-COPY --chown=${BAZEL_USER}:${BAZEL_USER} files/config.json ${BAZEL_USER_HOME}/.docker/config.json
+COPY --chown=bazel:bazel files/config.json ${BAZEL_USER_HOME}/.docker/config.json
```

Comment:
Should the user and group just be hardcoded everywhere for now since bazel and bazel need to exist for the chown?

Reply:
I don't think so. Interpolation works everywhere else and can be used for chown in the next version of docker.

Reply:
My point was that passing anything other than bazel for BAZEL_USER into this dockerfile will break the build, so it is currently an unusable option.